backend/src/main/java/de/jannisaziz/backend/user/UserDTO.java

```diff
@@ -0,0 +1,33 @@
+package de.jannisaziz.backend.user;
+
+import de.jannisaziz.backend.game.Game;
+import de.jannisaziz.backend.game.Review;
+import de.jannisaziz.backend.game.SavedGame;
+import lombok.*;
+import org.springframework.data.annotation.Id;
+import java.util.List;
+
+@Builder
+@Getter
+@Setter
+public class UserDTO {
```

Comment:
What is this dto used for? Has a lot of private data, why does the user need to see his or others oauthConnections?